### PR TITLE
added page /embed/white-labelling

### PR DESCRIPTION
**Pull request by maspio**

### CODE REVIEW

File: docs/embed/white-labelling.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# White Labelling
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:1]:
```suggestion
# White labelling
```

File: docs/embed/white-labelling.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+To white label n8n you customize the frontend styling and assets to match your brand identity. The process involves changing n8n's source code [github.com/n8n-io/n8n](https://github.com/n8n-io/n8n), specifically two packages:
+
+* [packages/design-system](https://github.com/n8n-io/n8n/tree/master/packages/design-system) - n8n's [storybook](https://storybook.js.org/) design system with CSS styles and Vue.js components
+* [packages/editor-ui](https://github.com/n8n-io/n8n/tree/master/packages/editor-ui) - n8n's [Vue.js](https://vuejs.org/) frontend build with [Vite.js](https://vitejs.dev)
+
+## Prerequisite
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:8]:
```suggestion
## Prerequisites
```

File: docs/embed/white-labelling.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# White Labelling
+
+To white label n8n you customize the frontend styling and assets to match your brand identity. The process involves changing n8n's source code [github.com/n8n-io/n8n](https://github.com/n8n-io/n8n), specifically two packages:
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:3]:
```suggestion
White labelling n8n means customizing the frontend styling and assets to match your brand identity. The process involves changing two packages in n8n's source code [github.com/n8n-io/n8n](https://github.com/n8n-io/n8n){:target=_blank .external-link}:
```

File: docs/embed/white-labelling.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+* [packages/design-system](https://github.com/n8n-io/n8n/tree/master/packages/design-system) - n8n's [storybook](https://storybook.js.org/) design system with CSS styles and Vue.js components
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:5]:
```suggestion
* [packages/design-system](https://github.com/n8n-io/n8n/tree/master/packages/design-system){:target=_blank .external-link}: n8n's [storybook](https://storybook.js.org/){:target=_blank .external-link} design system with CSS styles and Vue.js components
```

File: docs/embed/white-labelling.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+* [packages/editor-ui](https://github.com/n8n-io/n8n/tree/master/packages/editor-ui) - n8n's [Vue.js](https://vuejs.org/) frontend build with [Vite.js](https://vitejs.dev)
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:6]:
```suggestion
* [packages/editor-ui](https://github.com/n8n-io/n8n/tree/master/packages/editor-ui){:target=_blank .external-link}: n8n's [Vue.js](https://vuejs.org/){:target=_blank .external-link} frontend build with [Vite.js](https://vitejs.dev){:target=_blank .external-link}
```

File: docs/embed/white-labelling.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+## Prerequisite
+
+You need the following installed on your development machine:
+
+--8<-- "_snippets/integrations/creating-nodes/prerequisites.md"
+
+Create a fork of [n8n's repository](https://github.com/n8n-io/n8n) and clone your new repository.
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:14]:
```suggestion
Create a fork of [n8n's repository](https://github.com/n8n-io/n8n){:target=_blank .external-link} and clone your new repository.
```

File: docs/embed/white-labelling.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+```shell
+npm install
+npm run build
+npm run start
+```
+
+Whenever you make changes you need to rebuild and restart n8n. While developing you can use `npm run dev` to automatically rebuild and restart n8n anytime you make code changes. 
+
+## Theme Colors
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:31]:
```suggestion
## Theme colors
```

File: docs/embed/white-labelling.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+
+To customize theme colors look into [packages/design-system](https://github.com/n8n-io/n8n/tree/master/packages/design-system) and start with:
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:33]:
```suggestion
To customize theme colors open [packages/design-system](https://github.com/n8n-io/n8n/tree/master/packages/design-system){:target=_blank .external-link} and start with:
```

File: docs/embed/white-labelling.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+
+- [packages/design-system/src/css/_tokens.scss](https://github.com/n8n-io/n8n/blob/master/packages/design-system/src/css/_tokens.scss)
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:35]:
```suggestion
- [packages/design-system/src/css/_tokens.scss](https://github.com/n8n-io/n8n/blob/master/packages/design-system/src/css/_tokens.scss){:target=_blank .external-link}
```

File: docs/embed/white-labelling.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+- [packages/design-system/src/css/_tokens.dark.scss](https://github.com/n8n-io/n8n/blob/master/packages/design-system/src/css/_tokens.dark.scss)
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:36]:
```suggestion
- [packages/design-system/src/css/_tokens.dark.scss](https://github.com/n8n-io/n8n/blob/master/packages/design-system/src/css/_tokens.dark.scss){:target=_blank .external-link}
```

File: docs/embed/white-labelling.md [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+At the top of `_tokens.scss` you will find `--color-primary` variables as HSL colors:
+
+```scss
+@mixin theme {
+	--color-primary-h: 6.9;
+	--color-primary-s: 100%;
+	--color-primary-l: 67.6%;
+```
+
+In the following example the primary color changed to <span style="color:#0099ff">#0099ff</span>. To convert to HSL you can use a [color converter tool](https://www.w3schools.com/colors/colors_converter.asp).
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:47]:
```suggestion
In the following example the primary color changes to <span style="color:#0099ff">#0099ff</span>. To convert to HSL you can use a [color converter tool](https://www.w3schools.com/colors/colors_converter.asp){:target=_blank .external-link}.
```

File: docs/embed/white-labelling.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+@mixin theme {
+	--color-primary-h: 204;
+	--color-primary-s: 100%;
+	--color-primary-l: 50%;
+```
+
+![Example Theme Color Customization](/_images/embed/white-label/color-transition.gif)
+
+!!! note
+    Similar CSS variables in `_tokens.dark.scss` for Dark mode are an upcoming feature that you can't toggle using n8n's UI yet.
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:59]:
```suggestion
    Similar CSS variables in `_tokens.dark.scss` for dark mode are an upcoming feature that you can't toggle using n8n's UI yet.
```

File: docs/embed/white-labelling.md [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+
+## Theme Logos
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:61]:
```suggestion
## Theme logos
```

File: docs/embed/white-labelling.md [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+
+To change the editor’s logo assets look into [packages/editor-ui/public](https://github.com/n8n-io/n8n/tree/master/packages/editor-ui/public) and replace:
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:63]:
```suggestion
To change the editor’s logo assets look into [packages/editor-ui/public](https://github.com/n8n-io/n8n/tree/master/packages/editor-ui/public){:target=_blank .external-link} and replace:
```

File: docs/embed/white-labelling.md [hunk 15]
```diff
@@ -0,0 +1,142 @@
...
+
+- favicon-16x16.png
+- favicon-32x32.png
+- favicon.ico
+- n8n-logo.svg
+- n8n-logo-collapsed.svg
+- n8n-logo-expanded.svg
+
+Replace these logo assets used in different Vue.js components like
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:72]:
```suggestion
Replace these logo assets. n8n uses them in Vue.js components, including:
```

File: docs/embed/white-labelling.md [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+
+* [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue) - top/left logo in the main sidebar
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:74]:
```suggestion
* [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue){:target=_blank .external-link}: top/left logo in the main sidebar.
```

File: docs/embed/white-labelling.md [hunk 17]
```diff
@@ -0,0 +1,142 @@
...
+* [Logo.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/Logo.vue) - Reused in other components
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:75]:
```suggestion
* [Logo.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/Logo.vue): reused in other components.
```

File: docs/embed/white-labelling.md [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+
+In the following example replace `n8n-logo-collapsed.svg` and `n8n-logo-expanded.svg` to update the main sidebar's logo assets.
+
+![Example Logo Main Sidebar](/_images/embed/white-label/logo-main-sidebar.png)
+
+If your logo assets require different sizing or placement you can customize SCSS styles at the bottom of [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue).
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:81]:
```suggestion
If your logo assets require different sizing or placement you can customize SCSS styles at the bottom of [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue){:target=_blank .external-link}.
```

File: docs/embed/white-labelling.md [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+	.icon {
+		height: 18px;
+		position: relative;
+		left: 6px;
+	}
+
+}
+```
+
+## Text Localization
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:108]:
```suggestion
## Text localization
```

File: docs/embed/white-labelling.md [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+
+To change all text occurrences like `n8n` or `n8n.io` to your brand identity you can customize n8n's english internationalization file:
+
+- [packages/editor-ui/src/plugins/i18n/locales/en.json](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/plugins/i18n/locales/en.json)
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:112]: delete

File: docs/embed/white-labelling.md [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+
+n8n uses the [Vue I18n](https://kazupon.github.io/vue-i18n/) internationalization plugin for Vue.js to translate the majority of UI texts. To search and replace text occurrences inside `en.json` you can use [Linked locale messages](https://kazupon.github.io/vue-i18n/guide/messages.html#linked-locale-messages).
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:114]:
```suggestion
n8n uses the [Vue I18n](https://kazupon.github.io/vue-i18n/){:target=_blank .external-link} internationalization plugin for Vue.js to translate the majority of UI texts. To search and replace text occurrences inside `en.json` you can use [Linked locale messages](https://kazupon.github.io/vue-i18n/guide/messages.html#linked-locale-messages){:target=_blank .external-link}.
```

File: docs/embed/white-labelling.md [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+
+In the following example add the `_brand.name` translation key to white label n8n's [AboutModal.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/AboutModal.vue).
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:116]:
```suggestion
In the following example add the `_brand.name` translation key to white label n8n's [AboutModal.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/AboutModal.vue){:target=_blank .external-link}.
```

File: docs/embed/white-labelling.md [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+	"_brand.name": "My Brand",
+	//replace n8n with link to _brand.name
+	"about.aboutN8n": "About @:_brand.name",
+	"about.n8nVersion": "@:_brand.name Version",
+}
+```
+
+![Example About Modal Localization](/_images/embed/white-label/about-modal.png)
+
+### Window Title
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:129]:
```suggestion
### Window title
```

File: docs/embed/white-labelling.md [hunk 24]
```diff
@@ -0,0 +1,142 @@
...
+
+To change n8n`s window title to your brand name look at
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:131]:
```suggestion
To change n8n's window title to your brand name, edit the following:
```

File: docs/embed/white-labelling.md [hunk 25]
```diff
@@ -0,0 +1,142 @@
...
+
+- [packages/editor-ui/index.html](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/index.html)
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:133]:
```suggestion
- [packages/editor-ui/index.html](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/index.html){:target=_blank .external-link}
```

File: docs/embed/white-labelling.md [hunk 26]
```diff
@@ -0,0 +1,142 @@
...
+- [packages/editor-ui/src/components/mixins/titleChange.ts](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/mixins/titleChange.ts)
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:134]:
```suggestion
- [packages/editor-ui/src/components/mixins/titleChange.ts](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/mixins/titleChange.ts){:target=_blank .external-link}
```

File: docs/embed/white-labelling.md [hunk 27]
```diff
@@ -0,0 +1,142 @@
...
+
+In the following example replace all occurrences of `n8n` and `n8n.io` with `My Brand`.
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:136]:
```suggestion
The following example replaces all occurrences of `n8n` and `n8n.io` with `My Brand`.
```

File: docs/embed/white-labelling.md [hunk 28]
```diff
@@ -0,0 +1,142 @@
...
+
+![Example Window Title Localization](/_images/embed/white-label/window-title.png)
```
Comment (StarfallProjects) [inline docs/embed/white-labelling.md:138]: Should we also tell them what to change to get this effect? I think perhaps there should be a code snippet here?

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Looks good! Most of my comments are making sure external links open in new tab, and getting headings to conform to the style guide.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Looks good! Most of my comments are making sure external links open in new tab, and getting headings to conform to the style guide.

File: docs/embed/white-labelling.md (hunk 20 above)
Comment (maspio) [reply docs/embed/white-labelling.md:112]: What do you mean by delete?

Comment (maspio) [reply docs/embed/white-labelling.md:112]: ah you want to remove the list, gotcha.

File: docs/embed/white-labelling.md (hunk 28 above)
Comment (maspio) [reply docs/embed/white-labelling.md:138]: I'll look into it.

File: docs/embed/white-labelling.md (hunk 20 above)
Comment (StarfallProjects) [reply docs/embed/white-labelling.md:112]: Yeah remove that line after the previous edit. A single bullet point looks odd.

File: docs/embed/white-labelling.md (hunk 28 above)
Comment (maspio) [reply docs/embed/white-labelling.md:157]: Added code sample.

**Pull request merged**